recipe/meta.yaml

```diff
@@ -31,12 +31,18 @@ requirements:
     - meson
     - ninja
     - pkg-config
+    - pthread-stubs
```

Comment:
Has something changed that forcing us to include `pthread-stubs` as a build depends? These functions should be included in the sysroot for the C compiler.